cmds/replay.c

```diff
@@ -461,6 +461,18 @@ static void print_task_newline(int current_tid)
 			print_args("\\\\x%02hhx", c);			\
 	} while (0)
 
+#define print_escaped_char(c)						\
+        do {								\
+                if (c == '\0')						\
+                        print_args("\\0");				\
+                else if (c == '\b')					\
+                        print_args("\\b");				\
+                else if (c == '\n')					\
+                        print_args("\\n");				\
+                else							\
+                        print_args("%c", c);				\
```

Comment:
I think it's better not to call `snprintf()` since it'd be just a simple copy.

Reply:
Can i use `pr_out` instead of `print_args` at line 473? If this method is valid, i think `print_args` of `print_json_escaped_char` function(current line 459) are also need to be changed.

Reply:
No, it's a string copy, I just want to something like this:

```c
#define print_char(c) \
{( args[n] = c; n++; len--; })
```